app/src/main/java/com/tomtom/sdk/examples/MainMenu.kt

```diff
@@ -13,47 +13,52 @@ package com.tomtom.sdk.examples
 import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.tomtom.sdk.examples.databinding.ActivityMainMenuBinding
 import com.tomtom.sdk.examples.maps.MapExamplesActivity
 import com.tomtom.sdk.examples.navigation.NavigationExamplesActivity
 import com.tomtom.sdk.examples.offline.OfflineExamplesActivity
 import com.tomtom.sdk.examples.routing.RoutingExamplesActivity
 import com.tomtom.sdk.examples.search.SearchExamplesActivity
-import com.tomtom.sdk.examples.usecase.BasicNavigationActivity
 import com.tomtom.sdk.examples.usecase.MapModeSelectionActivity
 
 
 class MainMenu : AppCompatActivity() {
+    private lateinit var binding: ActivityMainMenuBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main_menu)
+
+        binding = ActivityMainMenuBinding.inflate(layoutInflater)
+        val rootView = binding.root
+        setContentView(rootView)
     }
 
-    fun launchBasicDriving(button : android.view.View) {
+    fun launchBasicDriving(button: android.view.View) {
```

Comment:
Import `View` instead of using a whole package path

Reply:
done